flow/cmd/mirror_status.go

```diff
@@ -347,18 +347,14 @@ func (h *FlowRequestHandler) getQRepConfigFromCatalog(ctx context.Context, flowJ
 }
 
 func (h *FlowRequestHandler) isCDCFlow(ctx context.Context, flowJobName string) (bool, error) {
-	var query pgtype.Text
-	err := h.pool.QueryRow(ctx, "SELECT query_string FROM flows WHERE name = $1", flowJobName).Scan(&query)
+	var isCdc bool
+	err := h.pool.QueryRow(ctx, "select exists(select * from flows where name = $1 and isnull(query_string, '') = '')",
```

Comment:
nil: SQL formatting

Reply:
How would you format it?

Reply:
`SELECT exists(SELECT * FROM flows WHERE name=$1 and coalesce(query_string, '')='')"`

basically capitalize SQL keywords and lowercase functions and identifiers